webapp/app/routes.py

```diff
@@ -104,6 +104,22 @@ def load_user(user_id):
 
         return user
 
+    @app.before_request
+    def log_flask_login_session_info():
+
+        def is_user_in_database():
+            user = None
+            user_id = session.get('_user_id')
+            if user_id is not None and login_manager._user_callback is not None:
+                user = login_manager._user_callback(user_id)
+            return user is not None
+
+        current_app.logger.info('{' + f"'Session protection intact: {not login_manager._session_protection_failed()},"
```

Comment:
We should also log whether the session cookie is set:

```suggestion
        current_app.logger.info('{' + f"'Request has session cookie: {'session' in request.cookies},"
                                      f"'Session protection intact: {not login_manager._session_protection_failed()},"
```